packages/playground/wordpress/src/rewrite-rules.ts

```diff
@@ -0,0 +1,11 @@
+import type { RewriteRule } from '@php-wasm/universal';
+
+/**
+ * The default rewrite rules for WordPress.
+ */
+export const wordPressRewriteRules: RewriteRule[] = [
+	{
+		match: /^\/(.*?)(\/wp-(content|admin|includes).*)/g,
+		keep: /(\/wp-(content|admin|includes).*)/g,
+	},
```

Comment:
I'm not sure I follow the keep / remove dichotomy. Rewriting URLs typically involves a pattern to match on and a replacement string for that pattern, for example:

```
	{
		match: /^\/(.*?)(\/wp-(content|admin|includes).*)/g,
		replacement: '$2',
	}
```

Could we do that here? Or are there specific advantages of using the `keep` / `remove` patterns?

Reply:
Also, should the match be on `/^\/([^/]+)(\/wp-(content|admin|includes).*)/` ?

Reply:
I went in this direction mostly because of my limited regex knowledge, but I can try to make it work with `replacement`.

Reply:
Yes please ❤️